test/e2e/node/pod_resize.go

```diff
@@ -838,6 +1034,27 @@ func getNodeAllocatableAndAvailableMilliCPUValues(ctx context.Context, f *framew
 	return nodeAllocatableMilliCPU, nodeAvailableMilliCPU
 }
 
+// Calculate available memory. nodeAvailableMemory = nodeAllocatableMemory - sum(podAllocatedMemory)
+func getNodeAllocatableAndAvailableMemoryValues(ctx context.Context, f *framework.Framework, n *v1.Node) (int64, int64) {
```

Comment:
optional: you could deduplicate this with the above method by taking a resource name as a parameter

Reply:
mind if I do this in a follow-up? It was (slightly) less straightforward than I expected since for CPU we use millivalues and for memory we are just using the values as-is